components/CustomerInfo/styles.js

```diff
@@ -25,11 +33,52 @@ const CustomerInfoButtonContainer = styled.section`
   display: flex;
   margin-top: 30px;
   gap: 50px;
+  margin-bottom: 130px;
```

Comment:
You could combine two margin styles in one:
```suggestion
  margin: 30px 0 130px;
  gap: 50px;
```